src/server/game/invitesmanager.js

```diff
@@ -273,7 +273,8 @@ function areUnderMaintenance(ws) {
     gamemanager.printActiveGameCount();
     let message = allowinvites.message;
     const timeUntilRestart = getMinutesUntilRestart();
-    if (timeUntilRestart) message += ` Minutes: ${timeUntilRestart}`
+    // TODO: send the timeUntilRestart with the message somehow
+    // if (timeUntilRestart) message += ` Minutes: ${timeUntilRestart}`
```

Comment:
I think it's better to include the time until the server restarts with this message, even though it may not translated yet, they'd have no way of knowing how soon.

Reply:
Done!